src/06-blackjack/Blackjack/Blackjack.Core/Deck.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Blackjack.Core
+{
+    public class Deck
```

Comment:
In this whole class you are making an assumption that Deck can only be 52 cards of all 4 suits. What if the business requirements change that Blackjack is played with 36 cards (6 to A).. It should be someone elses job to construct the Deck.